src/index.ts

```diff
@@ -21,6 +22,9 @@ function checkDiskSpace(directoryPath: string, dependencies: Dependencies = {
 	pathSep: sep,
 	cpExecFile: execFile,
 }): Promise<DiskSpace> {
+
+	const cpExecFile = promisify(dependencies.cpExecFile)
```

Comment:
Maybe rename it to avoid confusion?

Something like `cpExecFileAsync`

Reply:
Is there a need to create that here instead of doing that just before the only async call?

Reply:
I thought it would be nice to declare it outside so it runs only once, but it seems meaningless since `checkWin32` will run only once anyway. I'll move them.